GitHubActions/build_postis.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+################################################################################
+#
+# This script downloads PostgreSQL from the official web site into ./workdir
+# then builds it.
+#
+# Usage: ./build_postgis.sh pg_version postgis_version
+#     pg_version is a PostgreSQL version to be installed like 16.0.
+#     postgis_version is a PostGIS version to be installed.
+#
+# Requirements
+# - be able to connect to the PostGIS official web site by wget.
+# - having superuser privileges
+#
+################################################################################
```

Comment:
I see that the code that you show is calling `build_postgis.sh`.  
I commented about `build_postis.sh`.  
Could you explain how you call `build_postis.sh` at that code?

Reply:
> Could you explain how you call `build_postis.sh` at that code?

In case of `postgis` testing mode there is `build_postis.sh` script calling. Otherwise this CI step will be void. Maybe I misunderstanding Your question, @son-phamngoc ?

Reply:
I have understood, there is a file with error in name. Deleted, @son-phamngoc , thanks.

Reply:
Thank you. Confirmed. Please help me to resolve this comment.